README.md

```diff
@@ -40,7 +40,7 @@ Older than 1.0.3                      | [Upgrade to 1.0.3]    |
 At least 1.0.3 but older than 1.1.0   | [Upgrade to 1.1.0]    |
 At least 1.1.0 but older than 5.0.0   | [Upgrade to 5.0.0]    |
 At least 5.0.0 but older than 5.0.2   | [Upgrade to 5.0.2]    | For MySQL/MariaDB only
-At least 5.0.2 but older than 7.0.0   | [Upgrade to 7.0.0]    | For PostgreSQL only
+At least 5.0.2 but older than 7.0.0   | [Upgrade to 7.0.0]    | For all database types
```

Comment:
I suggest that the comment is empty when all database engines are affected.

Reply:
done